dces-drc-integration/src/integrationTest/java/uk/gov/justice/laa/crime/dces/integration/client/TestDataClient.java

```diff
@@ -1,33 +1,69 @@
 import org.springframework.web.service.annotation.PutExchange;
 import uk.gov.justice.laa.crime.dces.integration.maatapi.MaatApiClientFactory;
 import uk.gov.justice.laa.crime.dces.integration.maatapi.client.MaatApiClient;
 import uk.gov.justice.laa.crime.dces.integration.model.external.ConcorContributionResponseDTO;
+import uk.gov.justice.laa.crime.dces.integration.model.external.FdcContribution;
+import uk.gov.justice.laa.crime.dces.integration.model.external.FdcItem;
+import uk.gov.justice.laa.crime.dces.integration.model.external.UpdateFdcContributionRequest;
+import uk.gov.justice.laa.crime.dces.integration.model.external.UpdateRepOrder;
+import uk.gov.justice.laa.crime.dces.integration.model.external.CreateFdcContributionRequest;
 import uk.gov.justice.laa.crime.dces.integration.model.external.ContributionFileErrorResponse;
 import uk.gov.justice.laa.crime.dces.integration.model.external.ContributionFileResponse;
 import uk.gov.justice.laa.crime.dces.integration.model.external.UpdateConcorContributionStatusRequest;
 
 import java.util.List;
 
-@HttpExchange("/debt-collection-enforcement")
 public interface TestDataClient extends MaatApiClient {
-    @PutExchange("/concor-contribution-status")
+    @PutExchange("/debt-collection-enforcement/concor-contribution-status")
     @Valid
     List<Integer> updateConcorContributionStatus(@RequestBody UpdateConcorContributionStatusRequest updateConcorContributionStatusRequest);
 
-    @GetExchange("/concor-contribution/{id}")
+    @GetExchange("/debt-collection-enforcement/concor-contribution/{id}")
     @Valid
     ConcorContributionResponseDTO getConcorContribution(@PathVariable Integer id);
 
+    @PostExchange("/debt-collection-enforcement/fdc-items")
+    @Valid
+    void createFdcItems(@Valid @RequestBody final FdcItem fdcItemDTO);
+
+    @DeleteExchange("/debt-collection-enforcement/fdc-items/fdc-id/{fdcId}")
+    @Valid
+    void deleteFdcItems(@NotNull @PathVariable final Integer fdcId);
+
+    @DeleteExchange("/assessment/rep-orders/cc-outcome/rep-order/{repId}")
+    @Valid
+    void deleteCrownCourtOutcomes(@PathVariable Integer repId);
+
+    @PutExchange("/assessment/rep-orders")
+    @Valid
+    void updateRepOrderSentenceOrderDate(@RequestBody UpdateRepOrder updateRepOrder);
+
+    @PostExchange("/debt-collection-enforcement/fdc-contribution")
+    @Valid
+    FdcContribution createFdcContribution(@RequestBody CreateFdcContributionRequest fdcContribution);
+
+    @PatchExchange("/debt-collection-enforcement/fdc-contribution")
+    @Valid
+    int updateFdcContribution(@RequestBody UpdateFdcContributionRequest fdcContribution);
+
+    @GetExchange("/assessment/rep-orders")
+    @Valid
+    List<Integer> getRepOrders(@RequestParam int delay, @RequestParam String dateReceived, @RequestParam int numRecords, @RequestParam boolean fdcDelayedPickup, @RequestParam boolean fdcFastTrack);
+
     @GetExchange("/contribution-file/{contributionFileId}")
```

Comment:
Shouldn't the /contribution-file/ clients be moved to /debt-collection-enforcement/contribution-file/ - or has maat-cd-api changed too?

Reply:
Good spot, @dconneely! This came from a merge from main and I didn't realise that the `/debt-collection-enforcement` used to be the default root of all endpoints in this class which I had to remove as part of my changes. I'll add that in the next commit.